### DISCUSSION TITLE
Not allowed to load local resource

### DISCUSSION THREAD

**Discussion by nottraced** [Deluze/electron-vue-template#46] [Q&A] [answered]
Hello. I'm new to Electron. I would like to ask you how I could solve my problem. When launching the unpacked application via Exe, the home page (index.html ) is being loaded. But as soon as I try to switch to other html, an error appears in the console. (There is no error in the unconverted dev version of the project)

`Not allowed to load local resource: file:///D:/Programming/electron-vue-template-master/dist/win-unpacked/resources/app.asar/renderer/login.html`

**Comment by Deluze** [comment, marked as answer]
Thanks for opening up a discussion!

It's hard to tell what's going wrong without seeing your exact config and directory structure. I can give a guess on what's happening, if you haven't altered any configs:

By default, everything in `<root>/src` is being ignored by Electron Builder, this is because we need to pack Electron with transpiled JavaScript files instead of TypeScript / Vue files. These transpiled (build) files can be found in `<root>/build` after compilation. The reason why it works on dev is because Vite's dev server is able to serve the TypeScript files in `<root>/src` directly as JavaScript.

Normally, a Vue project only has 1 entry point / html file, hence your `login.html` is not being built. I can give you 2 suggestions for now:

* Look into Vue Router: You'd be able to create a separate `Login` route, in a `Login.vue` template. Any Vue package will work just fine with this template.
* Add your `login.html` file to `electron-builder.json` and copy it over to `renderer` directory while packaging. You can use[ this example](https://github.com/Deluze/electron-vue-template/blob/master/electron-builder.json#L21).

I heavily recommend installing Vue Router, as you can super easily define routes there :)
